### DISCUSSION TITLE
Request: Day & Night Cycle

### DISCUSSION THREAD

**Discussion by MrGuanaco** [runelite/runelite#18008] [Ideas]
Hello everyone.

Me and a group of friends are wanting to make a youtube video of OSRS following the 100 Days theme. We are looking for someone to make a Day and Night cycle for Runelite with adjustable day, night cycles and if possible add dawn and dusk. Would something like this be possible?

**Comment by YvesW** [comment]
Duplicate of https://github.com/runelite/runelite/discussions/15388
